download_cli.sh

```diff
@@ -127,19 +127,28 @@ if [[ ":$PATH:" != *":$GOOSE_BIN_DIR:"* ]]; then
 fi
 
 # --- 8) Auto-configure Goose (Optional) ---
-CONFIG_ARGS="-n default"
+CONFIG_ARGS=""
 if [ -n "${GOOSE_PROVIDER:-}" ]; then
   CONFIG_ARGS="$CONFIG_ARGS -p $GOOSE_PROVIDER"
 fi
 if [ -n "${GOOSE_MODEL:-}" ]; then
   CONFIG_ARGS="$CONFIG_ARGS -m $GOOSE_MODEL"
 fi
 
-echo ""
-echo "Configuring Goose with: '$CONFIG_ARGS'"
-echo ""
-"$GOOSE_BIN_DIR/$OUT_FILE" configure $CONFIG_ARGS
+# --- 8) Auto-configure Goose (Optional) ---
+CONFIG_ARGS=""
+[ -n "${GOOSE_PROVIDER:-}" ] && CONFIG_ARGS="$CONFIG_ARGS -p $GOOSE_PROVIDER"
+[ -n "${GOOSE_MODEL:-}" ] && CONFIG_ARGS="$CONFIG_ARGS -m $GOOSE_MODEL"
 
-echo ""
-echo "Goose installed successfully! Run '$OUT_FILE session' to get started."
+if [ -n "${GOOSE_PROVIDER:-}" ] || [ -n "${GOOSE_MODEL:-}" ]; then
```

Comment:
let's confirm the quarantine doesn't apply when curled in a developer setup, but assuming it doesn't i do still kind of like having configure auto run? we do have added a good number of tooltips suggesting configure elsewhere tho so this works

Reply:
i reverted and just removed the '-n default'. will test this out in the Canary release